java/team25core/DeadReckonTask.java

```diff
@@ -390,8 +390,8 @@ public boolean timeslice()
             break;
         }
 
-        robot.telemetry.addData("Segment: ", num);
-        robot.telemetry.addData("State: ", segment.state.toString());
+//        robot.telemetry.addData("Segment: ", num);
+//        robot.telemetry.addData("State: ", segment.state.toString());
```

Comment:
Either remove entirely or uncomment commented out software.  Please scan the entire diff, I won't comment for each instance.